src/ui/scss/component/_header.scss

```diff
@@ -2,108 +2,81 @@
   position: fixed;
   width: 100%;
   height: var(--header-height);
-  align-items: center;
-  background-color: $lbry-white;
   display: flex;
   justify-content: space-between;
-  z-index: 2;
+  z-index: 2; // Main content uses z-index: 1, this ensures it always scrolls under the header
```

Comment:
Not necessary to adjust in this PR, but it can be good to make your z-index values SASS variables for better clarity.